src/internal/operations/backup.go

```diff
@@ -292,12 +292,20 @@ func (op *BackupOperation) do(
 // checker to see if conditions are correct for incremental backup behavior such as
 // retrieving metadata like delta tokens and previous paths.
 func useIncrementalBackup(sel selectors.Selector, opts control.Options) bool {
-	// Delta-based incrementals currently only supported for Exchange
-	if sel.Service != selectors.ServiceExchange {
+	enabled := !opts.ToggleFeatures.DisableIncrementals
+
+	switch sel.Service {
+	case selectors.ServiceExchange:
+		return enabled
+
+	case selectors.ServiceOneDrive:
+		// TODO(ashmrtn): Remove the && part once we support permissions and
+		// incrementals.
+		return enabled && !opts.ToggleFeatures.EnablePermissionsBackup
```

Comment:
Permissions backup should not be a problem now.

```suggestion
	case selectors.ServiceOneDrive:
		return enabled
```

Reply:
There is still one caveat even though delta for permissions now works. If they try to run a delta update without permissions, we will run into issues where only half of the items will have permissions.

Reply:
hmmm this does raise a broader question of how we'd handle it normally though. I don't think there's really a good way for us to do that right now. Either we're going to have to keep the permissions backups and no-permissions backups separate and not the no-permissions backups act as bases for the permissions backups or we're always going to have to force backing up permissions. Otherwise I just don't think there's a safe way to handle incrementals with both of them

re: eventually enabling permissions backup when we have incrementals, this may push us towards adding something else that lets us tell kopia a file isn't cached (#2361). At least that would solve things the first time around... maybe?

Reply:
Flag for enabling/disabling permissions backup is only temporary, once it is outside of experimental, permissions are always backed up.

Reply:
are we going to allow users to pass a flag saying not to backup permissions? If we do then we're kind of back to square one I think :/

Reply:
As of now yes, but we will be removing the option to not backup permissions before we "release" permissions backup.

Reply:
alright, I think we have a reasonable path forward then. I think something like #2361 will allow us to pull in permissions for a backup that doesn't have them so we can still do incrementals as well. Once permissions is enabled for good we won't have to worry about which base we select because they'll always be the same